File: Source/typescript/backend/resources/EventStoreConfiguration.ts
// Copyright (c) Dolittle. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.

import { ResourceConfiguration } from './ResourceConfiguration';

/**
 * Represents the configuration for the Dolittle Event Store
 */
export class EventStoreConfiguration extends ResourceConfiguration {
    servers!: string[];
    database!: string;
}

File: Source/DotNET/Backend/Resources/EventStoreConfiguration.cs
// Copyright (c) Dolittle. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.

namespace Dolittle.Vanir.Backend.Resources
{
    public class EventStoreConfiguration : ResourceConfiguration
    {
        public string[] Servers { get; set; }
        public string Database { get; set; }
    }
}
